src/NuGet.Core/NuGet.Commands/CommandRunners/PackCommandRunner.cs

```
@@ -979,9 +978,17 @@ internal static void ExcludeFilesForLibPackage(ICollection<IPackageFile> files)
             PathResolver.FilterPackageFiles(files, file => file.Path, _libPackageExcludes);
         }
 
-        internal static void ExcludeFilesForSymbolPackage(ICollection<IPackageFile> files)
+        internal static void ExcludeFilesForSymbolPackage(ICollection<IPackageFile> files, SymbolPackageFormat symbolPackageFormat)
         {
             PathResolver.FilterPackageFiles(files, file => file.Path, _symbolPackageExcludes);
+            if(symbolPackageFormat == SymbolPackageFormat.Snupkg)
+            {
+                var toRemove = files.Where(t => !string.Equals(Path.GetExtension(t.Path), ".pdb", StringComparison.OrdinalIgnoreCase)).ToList();
```

We should log some warning when we're filtering these files

---

no, this is by design, so no need to log warnings.

---

I know this is by design but it feels weird to silently ignore these